java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -225,4 +226,25 @@ public interface ListBaseCommands {
      * }</pre>
      */
     CompletableFuture<String[]> rpopCount(String key, long count);
+
+    /**
+     * Inserts <code>element</code> in the <code>list</code> at key either before or after the <code>
```

Comment:
Wrap key with code please. I don't think you need to wrap list as you are referring to the redis object not java.